custom_components/ha_kia_hyundai/sensor.py

```diff
@@ -122,6 +128,30 @@ class KiaSensorEntityDescription(SensorEntityDescription):
     ),
 )
 
+SEAT_SENSOR_DESCRIPTIONS: Final[tuple[KiaSensorEntityDescription, ...]] = (
+    KiaSensorEntityDescription(
+        key="heated_driver_seat",
```

Comment:
maybe just driver_seat instead of heated_ since it can heat and cool? or climate_driver_seat?